src/operations/blas3/gemm_local.hpp

```diff
@@ -138,6 +139,8 @@ class Gemm<input_t, output_t, DoubleBuffer, NbcA, NbcB, ClSize, TileType,
   static_assert(cl_elems % packetize_t::packet_size == 0,
                 "Cache line size must be a multiple of packet_size");
 
+  // TODO : assert strides are multiples of matrices sizes or = 0
```

Comment:
I don't think we could add these `static_asserts` here as the `strides` are all runtime parameters.

Reply:
Comment removed as well.